src/cubi_tk/sodar/ingest_data.py

```diff
@@ -534,9 +457,10 @@ def execute(self) -> typing.Optional[int]:
             return res
 
         logger.info("Starting cubi-tk sodar {}", self.command_name)
-        logger.info("args: {}", self.args)
-
-        lz_uuid, transfer_jobs = self.build_jobs()
+        print_args(self.args)
+        irods_hash_scheme = iRODSCommon(sodar_profile=self.args.config_profile).irods_hash_scheme()
```

Comment:
not 100% happy about having creating a new instance here.
Maybe consider adding the option to instantiate iRODSTransfer before adding all the jobs to it?
(don't do this now, maybe a make an issue?)